main.js

```diff
@@ -1,61 +1,55 @@
 const books = document.getElementById('books');
-let titulo = document.getElementById('titulo');
-let autor = document.getElementById('autor');
+const titulo = document.getElementById('titulo');
+const autor = document.getElementById('autor');
 const addBtn = document.getElementById('btn');
 
-let dataBooks = [];
-
 if (localStorage.getItem('book')) {
   dataBooks = JSON.parse(localStorage.getItem('book'));
 }
 
 class Libro {
   constructor(title, author) {
+    this.dataBooks = [title, author];
     this.title = title;
     this.author = author;
   }
-}
 
-function deleteBook(index, libro) {
-  libro.remove();
-  dataBooks.splice(index, 1);
-  localStorage.setItem('book', JSON.stringify(dataBooks));
+  deleteBook(index) {
+    dataBooks.splice(index, 1);
```

Comment:
You defined a class variable `dataBooks` in Libro class, you should always use `this` syntax when you try to call the class variable in a class method: `this.dataBooks.splice()`.Check out these links: [classes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes) and [constructor ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/constructor) to learn more about classes and constructors in JavaScript.